io.github.arunsivaramanneo.GPUViewer.yml

```diff
@@ -0,0 +1,119 @@
+id: io.github.arunsivaramanneo.GPUViewer
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: 3.30
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+command: gpu-viewer
+cleanup:
+  - "*.a"
+  - "*.la"
+  - /include
+  - /lib/pkgconfig
+modules:
+  - name: gpu-viewer
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/{bin,share}
+      - cp -a . /app/share/gpu-viewer
+      - install -Dm755 gpu-viewer.sh /app/bin/gpu-viewer
+      - install -Dm644 Images/GPU_Viewer.png /app/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm644 gpu-viewer.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - desktop-file-edit --remove-key=Terminal  --remove-key=version /app/share/applications/${FLATPAK_ID}.desktop
+      - desktop-file-edit --set-key=Terminal --set-value=false /app/share/applications/${FLATPAK_ID}.desktop
+      - desktop-file-edit --set-key=Exec --set-value=gpu-viewer /app/share/applications/${FLATPAK_ID}.desktop
+      - desktop-file-edit --set-key=Icon --set-value=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 -t /app/share/appdata ${FLATPAK_ID}.appdata.xml
+    sources:
+      - type: archive
+        url: "https://github.com/arunsivaramanneo/GPU-Viewer/archive/v1.14.tar.gz"
+        sha256: e81804dca0576cb7d3ccda78f3b34aeb3b16c99110a41b60b0c661737dbc609b
+      - type: script
```

Comment:
Why is this needed? You could just link the GPUViewer.py to /app/bin  & use it in `command` property (will need to update the desktop file too)

Reply:
No, I can't. The app hardcodes some relative paths, e.g. [here](/arunsivaramanneo/GPU-Viewer/blob/64f1e671c09350e3128eb07c04cdec2f5421d811/Files/Const.py).